trackintel/analysis/metrics.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+from tqdm import tqdm
+
+from trackintel.geogr import point_haversine_dist, check_gdf_planar
+
+
+def radius_gyration(gdf, method="count", print_progress=False):
+    """
+    Radius of gyration for individual users.
+
+    Parameters
+    ----------
+    gdf : Geodataframe
+        GeoDataFrame with column "user_id" and Point geometry.
+
+    method: string, {"count", "duration"}
+        Weighting for center of mass and average distance calculation.
+        - "count": assigns each Point the same weight of 1.
+        - "duration": assigns each Point a weight based on duration.
+           Additionally needs Timedelta column "duration" in gdf.
+
+    print_progress: bool, default False
+        Show per-user progress if set to True.
+
+    Returns
+    -------
+    Series
+        Radius of gyration for individual users.
+
+    References
+    ----------
+    [1] Gonzalez, M. C., Hidalgo, C. A., & Barabasi, A. L. (2008).
+    Understanding individual human mobility patterns. Nature, 453(7196), 779-782.
+
+    """
+    if method not in ["count", "duration"]:
+        raise ValueError(f'Method unknown. Should be on of {{"count", "duration"}}. You passed "{method}"')
+
```

Comment:
Rg also only makes sense if the sps are in the visited sequence. Since trackintel requires time stamp, I think we should add a `sort_values(by=["user_id", "started_at"])` here.

Reply:
Why should we order it? I don't understand the difference in just leaving it as it is.

Reply:
Second thought: we actually do not need to sort for rg. Sorry, Ignore this comment.